requirements.in

```diff
@@ -9,21 +9,26 @@ django_csp
 django-environ
 django-filter
 django-redis
+django-rq
 django-storages
 factory_boy
 Faker
 filebrowser-safe
 future
 gunicorn
+pygit2==1.0.3
 python-slugify
+# Need to install a pre-release in order to get the RedLock.locked() method
+git+git://github.com/glasslion/redlock.git@master#egg=redlock
```

Comment:
This is not something we want to rely on, it's trivial for this to become malicious if there isn't a version or exact commit we can point to.

Reply:
This is a backport of what we’re already using on Donate https://github.com/mozilla/donate-wagtail/blob/master/requirements.in#L20, so it’s something we will want to fix there too.

@kaedroho would you have an alternative to this? Maybe we don’t need a prerelease anymore?

Reply:
Last commit to redlock was 2 years ago. If the pypi version doesn't work, let's absolutely link to the commit hash as a way to "lock" this package.

Reply:
If it's that old, I'd also worry about undocumented security vulnerabilities... is redlock our only option?